### PR TITLE
add task

### PR DESCRIPTION
**Pull request by vmisai**
[DEMO LINK](https://vmisai.github.io/layout_catalog/)
[TEST REPORT LINK](https://vmisai.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,230 @@
...
+    display: block;
+    height: 40px;
+    width: 40px;
+
+    &-image {
+      height: 100%;
+      width: 100%;
+    }
+  }
+}
```
Comment (sTorba24) [inline src/styles/main.scss:40]: move each BEM block to separated file

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,230 @@
-body {
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+$white-color: #fff;
+$border-color: #f3f3f3;
+$box-shadow-color: rgba(0, 0, 0, 0.05);
+$title-hover-color: #34568b;
+$animation-duration: 300ms;
```
Comment (sTorba24) [inline src/styles/main.scss:8]: move it to variables file

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,230 @@
...
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```
Comment (sTorba24) [inline src/styles/main.scss:218]: background shorthand property can be use

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Well done! Lets improve a little

File: src/index.html [hunk 4]
```diff
@@ -3,10 +3,332 @@
...
+    </header>
+
+    <main>
+      <div class="cards">
+        <div data-qa="card" class="card cards__card">
+          <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac"
+          >
```
Comment (etojeDenys) [inline src/index.html:67]:
```suggestion
          <img
            class="card__image"
            src="images/imac.jpeg"
            alt="imac"
          >
```

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**